cip/CIP2016-06-14-Redefine-comparability-and-equality-as-well-as-orderability-and-equivalence.adoc

```diff
@@ -0,0 +1,367 @@
+= CIP2016-06-14 - Redefine Comparability and Equality, as well as Orderability and Equivalence
```

Comment:
I would like it if we could frame these documents as part of the specification, and not just enhancement proposals that you look at once and then throw away. In that light this title is wrong, this document is not (/ should not be) about _re_-defining Comparability, Equality, Orderability, and Equivalence, it is about _the definition_ of those things.

The PR is about re-defining these things, but the document is about the (new) definition of them.